PowerShellGet/PSModule.psm1

```diff
@@ -4906,6 +4906,16 @@ function Test-ScriptFileInfo
             return
         }
 
+        if ($PSScriptInfo.Version -match '-')
+        {
+            # Version contains PreRelease string.  Break apart and validate separately.
+            $indexOfFirstHyphen = $PSScriptInfo.Version.IndexOf('-')
+            $versionSubstring = $PSScriptInfo.Version.Substring(0, $indexOfFirstHyphen)
+            $preReleaseSubstring = $PSScriptInfo.Version.Substring($indexOfFirstHyphen+1)
+
+            $preReleaseIsValid = Validate-PreReleaseString -Version $versionSubstring -PreRelease $preReleaseSubstring -InfoObject $PSScriptInfo

```

Comment:
>$preReleaseIsValid [](start = 12, length = 18)

It looks like this variable is assigned but not used. If Validate-PrereleaseString is throwing a terminating exception for invalid case, it is not required to return true/false.